pkg/index/job/correction/service/corrector.go

```diff
@@ -89,14 +90,17 @@ func New(cfg *config.Data, discoverer discoverer.Client) (Corrector, error) {
 	}, nil
 }
 
-func (c *correct) Start(ctx context.Context) (<-chan error, error) {
-	// set current time to context
-	ctx = embedTime(ctx)
-
+func (c *correct) StartMonitoring(ctx context.Context) (<-chan error, error) {
 	dech, err := c.discoverer.Start(ctx)
 	if err != nil {
 		return nil, err
 	}
+	return dech, nil
```

Comment:
It would be better to implement it with one line without error handling!
```suggestion
	return c.discoverer.Start(ctx)
```